README.md

```diff
@@ -76,6 +76,9 @@ The current Dockerfile and docker-compose were closely copied from [the Docker g
 
 **Note:** If you are using Windows, you can run `bin/dbuild` using Git Bash.
 
+**Note:** If you are using a Mac with an Apple Silicon processer (M1, M2, etc) you will need to enable the following option in Docker's settings:
```

Comment:
Weird, I have an M2 Mac and I don't think I've had to do any of this

Reply:
Interesting. My main laptop is an M1. I have access to an M2 so I'll try out what's in `master` on that machine from scratch and see what happens.

Reply:
As per the commit message, without this setting turned on I got this error when running any `rake` commands:

```
  Errno::ENOSYS: Function not implemented - Failed to initialize inotify
  /usr/local/bundle/gems/rb-inotify-0.10.1/lib/rb-inotify/notifier.rb:69:in `initialize'
```

Reply:
Can confirm, on a separate M2 machine I see the exact same behaviour. Building the image from `master` fails on the Google Chrome step.

In my branch, disabling the Rosetta setting in Docker (the default) stops `rake` commands from running:

<details>
<summary><code>./bin/dbuild</code></summary>

```
➜  planner git:(apple-silicon-support) ./bin/dbuild
[+] Building 0.7s (13/13 FINISHED
<snip>
[+] Creating 1/0
 ✔ Container planner-db-1  Running
/usr/local/bundle/gems/net-protocol-0.2.1/lib/net/protocol.rb:68: warning: already initialized constant Net::ProtocRetryError
/usr/local/lib/ruby/2.7.0/net/protocol.rb:66: warning: previous definition of ProtocRetryError was here
/usr/local/bundle/gems/net-protocol-0.2.1/lib/net/protocol.rb:214: warning: already initialized constant Net::BufferedIO::BUFSIZE
/usr/local/lib/ruby/2.7.0/net/protocol.rb:206: warning: previous definition of BUFSIZE was here
/usr/local/bundle/gems/net-protocol-0.2.1/lib/net/protocol.rb:541: warning: already initialized constant Net::NetPrivate::Socket
/usr/local/lib/ruby/2.7.0/net/protocol.rb:503: warning: previous definition of Socket was here
[SKYLIGHT] [5.3.4] Unable to start, see the Skylight logs for more details
********************************************************************************
WARNING: Missing consumer key or secret. First, register an app with Github
Check README.md for instructions on how to set up Github Authentication.
********************************************************************************
rake aborted!
Errno::ENOSYS: Function not implemented - Failed to initialize inotify
/usr/local/bundle/gems/rb-inotify-0.10.1/lib/rb-inotify/notifier.rb:69:in `initialize'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/linux.rb:31:in `new'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/linux.rb:31:in `_configure'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/base.rb:45:in `block in configure'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/base.rb:40:in `each'
/usr/local/bundle/gems/rb-inotify-0.10.1/lib/rb-inotify/notifier.rb:69:in `initialize'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/linux.rb:31:in `new'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/linux.rb:31:in `_configure'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/base.rb:45:in `block in configure'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/base.rb:40:in `each'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/base.rb:40:in `configure'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/adapter/base.rb:63:in `start'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/listener.rb:68:in `block in <class:Listener>'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/fsm.rb:121:in `instance_eval'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/fsm.rb:121:in `call'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/fsm.rb:91:in `transition_with_callbacks!'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/fsm.rb:57:in `transition'
/usr/local/bundle/gems/listen-3.1.5/lib/listen/listener.rb:91:in `start'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/evented_file_update_checker.rb:96:in `boot!'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/evented_file_update_checker.rb:67:in `initialize'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/i18n_railtie.rb:61:in `new'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/i18n_railtie.rb:61:in `initialize_i18n'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/i18n_railtie.rb:19:in `block in <class:Railtie>'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/lazy_load_hooks.rb:69:in `block in execute_hook'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/lazy_load_hooks.rb:62:in `with_execution_control'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/lazy_load_hooks.rb:67:in `execute_hook'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/lazy_load_hooks.rb:52:in `block in run_load_hooks'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/lazy_load_hooks.rb:51:in `each'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/lazy_load_hooks.rb:51:in `run_load_hooks'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/application/finisher.rb:75:in `block in <module:Finisher>'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/initializable.rb:32:in `instance_exec'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/initializable.rb:32:in `run'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/initializable.rb:61:in `block in run_initializers'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/initializable.rb:60:in `run_initializers'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/application.rb:361:in `initialize!'
/planner/config/environment.rb:5:in `<main>'
/usr/local/bundle/gems/bootsnap-1.16.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/usr/local/bundle/gems/bootsnap-1.16.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/usr/local/bundle/gems/skylight-5.3.4/lib/skylight/probes.rb:166:in `require'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/dependencies.rb:291:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/dependencies.rb:257:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.8.1/lib/active_support/dependencies.rb:291:in `require'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/application.rb:337:in `require_environment!'
/usr/local/bundle/gems/railties-5.2.8.1/lib/rails/application.rb:520:in `block in run_tasks_blocks'
/usr/local/bundle/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:drop => db:load_config => environment
(See full trace by running task with --trace)
Built.
```
</details>

(Ignore the noise in the preamble about repeated constants and missing env file. They are separate problems)

Turning on the Rosetta setting allows the database seeding to complete.

I'm seeing the behaviour on two separate machines. It would be good to understand why you are able to build successfully on an Apple Silicon machine without any additional steps.

Reply:
Hey. Please disregard my comments. I forgot that I'm working on GitHub codespaces on my M2 Mac. I tried it on my local machine and saw the same errors you're seeing